### ISSUE TITLE
Video rerender when pressing play button

### ISSUE THREAD

**Issue by yoann84** [MetaLabs-inc/react-native-vimeo-iframe#4]
Hi,

I'vo got an issue when I attempt to play a video.
When I interact with the video (play, settings...) it makes the video rerender and the video doesn't start.
The onReady function which should log a message doesn't appear also.
I've got the image of the video so it's not related of the videoId itself (which also works in standard webview)

Here's the code which is very basic : 
```
export default function PlayerVideo({ navigation, route }) {

  const videoData = useSelector((state) => getItem(state, "videos", route?.params?.videoId));
  const videoId = React.useMemo(() => videoData?.code?.split("/")[2], [videoData?.code]);

  return (
    <SharedSafeArea bgColor="black">
      <View style={{ flex: 1 }}>
        <Vimeo
          videoId={videoId}
          onReady={() => console.log("Video is ready")}
          onPlay={() => console.log("Video is playing")}
          onPlayProgress={(data) => console.log("Video progress data:", data)}
          onFinish={() => console.log("Video is finished")}
          loop={false}
          autoPlay={false}
          controls={true}
          speed={false}
        />
      </View>
    </SharedSafeArea>
  );
}

```
**Note:** To make the package run without error at start I had to change the path in src/index.tsx to : 
`import webplayer from '../lib/template'`
```import { LayoutProps } from '../lib/types'```

I guess you're using path resolver but it is why some people run into issue of "./template missing file..."

**Comment by Jothinayagan** [comment]
I'm also facing exactly same!

**Comment by Jothinayagan** [comment]
@yoann84 Did you find any alter method?

**Comment by yoann84** [comment]
Yes I managed to make it work by changing some lines of code but it was still not working properly so I just give up that library and use react native webview instead with injected javascript.

**Comment by Marcoo09** [comment]
Fixed on #7
